tests/core/evaluation/test_marker_tracker_loader.py

```diff
@@ -20,6 +21,29 @@ def marker_trackerstore() -> TrackerStore:
     return store
 
 
+def test_load_sessions(tmp_path):
+    """Tests loading a tracker with multiple sessions."""
+    domain = Domain.empty()
+    store = SQLTrackerStore(domain, db=os.path.join(tmp_path, "temp.db"))
```

Comment:
btw, with paths, you can also do things like 
```suggestion
    store = SQLTrackerStore(domain, db= tmp_path / "temp.db")
```
(https://docs.python.org/3/library/pathlib.html#basic-use)